docs/components/operate/index.md

```diff
@@ -20,9 +20,9 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
 
 In addition to providing visibility into active and completed process instances, Operate also makes it possible to carry out key operations such as resolving [incidents](./userguide/resolve-incidents-update-variables.md), and updating process instance variables.
 
-[//]:# (What is the link to fix here?)
+[//]: # "What is the link to fix here?"
```

Comment:
This comment should actually be addressed. @christinaausley can you follow up here?

Reply:
Removed this in my latest commit.